src/nvim/api/ui_events.in.h

```diff
@@ -58,7 +58,8 @@ void set_title(String title)
   FUNC_API_SINCE(3);
 void set_icon(String icon)
   FUNC_API_SINCE(3);
-
+void highlight_info_set(Array highlights_changed)
+  FUNC_API_SINCE(3);
```

Comment:
Do you plan to use it in the TUI? Otherwise it should be REMOTE_ONLY and remove unnecessary boilerplate